scripts/promote-cherry-picks.ts

```diff
@@ -17,20 +17,23 @@ const {amp_version: ampVersion}: Args = yargs(process.argv.slice(2))
 
 const jobName = 'promote-cherry-pick.ts';
 
-function getOldAmpVersion(
+function getAmpVersionToCherrypick(
   ampVersion: string,
   currentVersions: Versions
 ): string {
-  const firstTen = ampVersion.slice(0, 10);
-  const oldVersion = Object.values(currentVersions).find(
-    (version) => version?.slice(2, 12) == firstTen
+  const cherryPicksCount = ampVersion.slice(-3);
+  const ampVersionWithoutCherryPicksCounter = ampVersion.slice(0, 10);
```

Comment:
Supernit: Instead of creating `ampVersionWithoutCherryPicksCounter` and `cherryPicksCount` again here, make them parameters of this function and pass them from your `createVersionsUpdatePullRequest` impl (you'll have to move their initialization above the call, obvs)